.github/workflows/continuous-integration.yml

```diff
@@ -39,13 +39,7 @@ jobs:
       matrix:
         os: [ubuntu, macos, windows]
         python: [3.6, 3.7, 3.8]
```

Comment:
Taking into account that `nox` is capable of running the tests on every Python version we want, do we really need to set Python versions on the `matrix`? Maybe we can save spare some CI time by reducing the number of virtual machines that we build on every run.

Reply:
I thought we might but this is related to why it's skipping your 3.6 build. Nox doesn't download new Python versions unless using `conda`. So it will only use what's in the system already. And I don't think we can set multiple Python's on Actions. So it's better to keep it this way. But the time saved by not using miniconda for everything is much larger than spinning up a vm.

Reply:
You convinced me! Let's keep it this way for now then!